presto-native-execution/scripts/setup-macos.sh

```diff
@@ -16,7 +16,7 @@ set -eufx -o pipefail
 # Run the velox setup script first.
 source "$(dirname "${BASH_SOURCE}")/../velox/scripts/setup-macos.sh"
 
-MACOS_DEPS="${MACOS_DEPS} bison gperf libsodium"
```

Comment:
Unrelated change? 

It would be great to split changes into separate commits with clear descriptions.

Reply:
This is also a cleanup. libsodium got moved to the velox script.
I can move it to a separate PR.

Reply:
Presto project supports multi-commit PRs. It would be best to split unrelated changes into separate commits within the same PR.

Reply:
Fixed